packages/yew/src/functional/hooks/use_reducer.rs

```diff
@@ -95,6 +95,40 @@ where
     }
 }
 
+#[cfg(feature = "nightly")]
+mod feat_nightly {
+    use super::*;
+
+    impl<T> FnOnce<(T::Action,)> for UseReducerHandle<T>
```

Comment:
Should this be implemented for `UseReducerHandle`?

Dispatch on `UseReducerHandle` implies a setter-like action where as `Deref` implies a reader-like behaviour.

Suppose `T` also implements `Fn(T::Action)`, then `handle()` and `(*handle)()` will actually dispatch to completely different functions. (This may be more likely to happen when `T::Action` is a primitive type like `u32`.)

In addition, if `UseReducerHandle` gets this behaviour, why not give this to `UseStateHandle` as well?

Reply:
I agree. I'll remove that